### ISSUE TITLE
How many active users?

### ISSUE THREAD

**Issue by brodybits** [react-native-community/react-native-tvos#15]
Survey from an outsider: I was wondering who is using this project and not in secret?

**Comment by NickAVolpe** [comment]
I have not contributed code to this project but I am actively using it. We are building a tvOS and Amazon FireStick app. I am a part of a small lean team inside a larger company. After a couple days of R&D we decided to go with this project as opposed to You.I TV. 

I appreciate all the hard work to keep this project going. If I can find ways to help contribute in the future I will.

**Comment by respectTheCode** [comment]
At Playlister our Apple TV app uses this project. We have 3 - 5 engineers working on the project and have been in production for about a year and a half. The rest of the platform is React and React Native so React Native was the natural choice for our Apple TV app. We recently moved from the core repo to this project.

**Comment by hufkens** [comment]
We build a TV App with React Native (TADAAM: https://tadaam.be). After they announced that the tvOS support could be dropped in the next releases we switched to this repo. 

The good news is that @douglowder did a great job setting all this up. One of our own custom components (the FocusGuide) was also added and merged. That would never have happened in the Core repo. So please contribute and help us out.

**Comment by zero298** [comment]
Also building OTT application with React Native.

**Comment by brodybits** [comment]
Thanks to all for the comments, nice to see that the tvOS support is alive and in active use. Any objections to keeping this issue open?

A quick disclaimer is that I am just an interested outsider in this project.

**Comment by dlowder-salesforce** [comment]
Yes I’ll keep this open!

**Comment by khat33b** [comment]
At JunctionTV, we have built the CONtv and Dove Channel apps using React Native for the last two years after switching from native. Now we are starting to build TV apps also. We will be using this repo for our TV apps.

Thanks @dlowder-salesforce and the contributors for all the hard work in maintaining this repo.

**Comment by ishaqashraf** [comment]
i am using this package and it's awesome. and i am building tvOS for my company

**Comment by Kazetsukai** [comment]
Yes many thanks to @dlowder-salesforce et al., we are using this repo to support AppleTV on our video-on-demand platform here at Shift72

**Comment by mehthabux** [comment]
🤚We are using this repo to develop an androidTv and tvOS app ,  we are developing an app similar to netflix, which includes many flatlists, video component and navigations. Till now i have suffered any issues.
Thanks to @dlowder-salesforce

**Comment by gaboro-moove-it** [comment]
I'm using this library to build an app to android tv an apple tv

**Comment by miblanchard** [comment]
The NFL app on tvOS, Android TV and FireTV are all built with this fork!

**Comment by GustavPS** [comment]
Using this to build an app line plex/emby for Android TV 😊

**Comment by samandarkuchkarov** [comment]
Using this to build android tv  project 😄

**Comment by esegebart** [comment]
> We build a TV App with React Native (TADAAM: https://tadaam.be). After they announced that the tvOS support could be dropped in the next releases we switched to this repo.
> 
> The good news is that @dlowder-salesforce did a great job setting all this up. One of our own custom components (the FocusGuide) was also added and merged. That would never have happened in the Core repo. So please contribute and help us out.

Wow this is amazing! This gives me hope.  I am trying to figure out how to get the fork using this project set up but your project looks amazing!

**Comment by douglowder** [comment]
@esegebart you can do

```sh
react-native init TestTVApp --template=react-native-tvos
```

and that will set up a new project using this repo.

This is from 2020, but the fundamentals discussed in this article are generally still up to date:

https://douglowdercode.medium.com/react-native-on-the-big-screen-part-1-introduction-c2734db18289

**Comment by esegebart** [comment]
> @esegebart you can do
> 
> ```shell
> react-native init TestTVApp --template=react-native-tvos
> ```
> 
> and that will set up a new project using this repo.
> 
> This is from 2020, but the fundamentals discussed in this article are generally still up to date:
> 
> https://douglowdercode.medium.com/react-native-on-the-big-screen-part-1-introduction-c2734db18289

Fantastic!!! I knew I had seen this page before but couldn't find it on here. I got it set up and about to give it a whirl! Thank you so much!

**Comment by avishka-codimite** [comment]
Hi 👋 , I recently started using this repo. This is great 😊 . I am building a TV app using this package. Sometimes when doing development work, problems arise and get stuck if there is no information to read. So I want to know, is there any forum where we can discuss issues and improvements related to this package? Please let me know.

**Comment by douglowder** [comment]
@avishka-codimite we can use Twitter for this, maybe the community https://twitter.com/i/communities/1509407040095068166 would be a good place to start. If there is enough interest, I can set up a discussion repo in the react-native-tvOS workspace.

**Comment by avishka-codimite** [comment]
@douglowder thank you. yeah, I think it better if have a discussion section 👍

**Comment by esegebart** [comment]
@douglowder @avishka-codimite I would be interested in a discussion section! I am stuck on a few things.

**Comment by nriccar** [comment]
Discussions were created by @douglowder on https://github.com/react-native-tvos/react-native-tvos/discussions
@esegebart Feel free to create a discussion and I'll be happy to help!

**Comment by esegebart** [comment]
@douglowder Thank you so much!!!
